_content/articles/sermon-on-the-letting-in-and-not-letting-in_ernst_waldschmidt.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Mahāmaudgalyāyana's Sermon on the Letting-in and Not Letting-in (of Sensitive Influences)"
+authors:
+  - "Ernst Waldschmidt"
+external_url: "https://journals.ub.uni-heidelberg.de/index.php/jiabs/article/view/8453/2360"
+drive_links:
+  - "https://drive.google.com/file/d/1sVIf327n20H0n_asX3wD4jPPYhJisvXS/view?usp=sharing"
+course: problems
+tags:
+  - sense-restraint
+year: 1978
+journal: jiabs
+publisher: Heidelberg University
+address: "Heidelberg, Germany"
+volume: 1
+number: 1
+pages: "25--33"
+---
+
+> "...if a monk avoids positive or negative inclinations when using his senses, Mara would not get access, would not get a hold."
+
+A brief description of Mahāmaudgalyāyana's sermon on sense-restraint. The author then shares and translates Sanskrit fragments of this sermon found in Turfan (Turpan).
```

Comment:
Let's add a link to its Pāli Parallel: SN 35.243 at the end of the first sentence.